swc-uxp-react-starter/README.md

```diff
@@ -0,0 +1,214 @@
+2. [Yarn package manager](https://yarnpkg.com/getting-started/install)
+3. UXP Developer Tool (UDT)
+4. UXP >= 7.2
+
+**Build and run**
+1. Start by installing the dependencies `yarn install`.
+2. Prepare the bundle using Webpack `yarn build`. You will notice a **dist** folder after this step.
+3. (Optional) `yarn watch` to automatically build the project every time you update a source file and `yarn start` to keep the plugin running and automatically build after every change.
+
+**Load the plugin into the application via UDT**
+1. Make sure the application is running and you can see it under 'Connected apps'.
+2. Click on 'Add Plugin' button and select the `manifest.json` of this plugin.
+3. Configure the `dist` folder of your plugin by using 'More' -> 'Advanced' option from the action menu `•••`
+4. Click on the ••• menu on the corresponding plugin row. Select 'Load' to view the plugin inside your application.
+5. (Optional) Select 'Watch' from plugin actions ••• to dynamically load the latest plugin changes. Note that a manifest change would need you to 'Unload' and 'Load' the plugin from scratch.
+
+
+<img width="800" alt="package.json with resolutions block" src="assets/load-plugin-from-developer-tools.png">
+
+<br></br>
+
+You should be able to see a banner in Photoshop plugin.
+<img width="350" alt="package.json with resolutions block" src="assets/menu-starter.png">
+
+
+
+## Add a new component
+You can use any of the supported components listed in [our docs](https://developer.adobe.com/photoshop/uxp/2022/uxp-api/reference-spectrum/swc) by following these steps:
+
+1. Use `yarn add` to install the components and its supported version.
+
+```javascript
+yarn add @swc-uxp-wrappers/link;
+yarn add @swc-react/link@0.14.1-react.3029;
+```
+
+**Note:** The UXP-SWC components are delivered via wrappers over specific SWC versions. For example, `@swc-uxp-wrappers/menu` is locked and wrapped on  **0.16.9** version of ```@spectrum-web-components/menu```. Thereby for React framework we will need to use ```@swc-react/menu:0.16.9-react.3029```, which is the react-wrapper closest to the SWC component v0.16.9, in the dependency block too. Also, please note that components like `icons`,`icons-workflow`, `icons-ui`, `theme`, `shared` , `base`,`styles` do NOT use wrappers and therefore must be directly consumed from `@spectrum-web-components` library npm.
+
+<img width="1125" alt="image2023-1-20_18-45-41" src="assets/package.png">
+
+
+2. Register the component in `App.js` file.
+
+```javascript
+import { Link } from "@swc-react/link";
+```
+
+3. Add the component in the same file.
+
+```html
+   This is a <Link href="#">example link</Link>.
+```
+
+4. Add the alias configuration in the Webpack config file.
+This project contains the configuration already. Notice the `@swc-uxp-wrappers/utils` entry in the package.json. This package delivers the `alias.js` file for all the supported components which is then imported in the `webpack.config.js` file as this.
```

Comment:
Do we need to include alias.js?

Reply:
It is included in @swc-uxp-wrappers/utils package

Reply:
Done